composer.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "diablomedia/gpcsv-generator",
+    "type": "library",
+    "description": "Library to assist with the generation of CashPro Global Payments GPCSV files",
+    "keywords": ["gpcsv"],
+    "homepage": "https://github.com/diablomedia/gpcsv-generator",
+    "license": "LGPL-3.0-or-later",
+    "authors": [
+        {
+            "name": "Ari Pringle",
+            "email": "ari@diablomedia.com",
+            "homepage": "https://diablomedia.com"
+        }
+    ],
+    "require": {
+        "ext-filter": "*",
+        "php": "^7.2"
```

Comment:
I believe this will prevent PHP 8 installs, which may or may not be what you want, just want to be sure. :)